modules/core/test/v2/unit/coins/ltc.ts

```diff
@@ -63,21 +63,24 @@ describe('LTC:', function() {
       version2Address.should.equal(bech32Address);
     });
 
-    it('upper case bech32 mainnet address', function() {
-      // bech32 addresses which are all upper case or all lower case are potentially valid,
-      // but mixed case is always invalid. Canonical bech32 addresses should be all lower case however
-      const bech32Address = 'LTC1QGRL8ZPNDSKLAA9SWGD5VEVYXMX5X63VCRL7DK4';
-      const newAddress = ltc.canonicalAddress(bech32Address);
-      newAddress.should.equal(bech32Address.toLowerCase());
+    it('non-lowercase bech32 mainnet address should fail', function () {
+      // bech32 addresses which are non-lowercase are invalid,
+      const bech32Address = 'ltc1QGRL8ZPNDSKLAA9SWGD5VEVYXMX5X63VCRL7DK4';
+      (() => ltc.canonicalAddress(bech32Address)).should.throw('invalid address');
     });
 
-    it('bech32 testnet address', function() {
+    it('lower case bech32 testnet address', function () {
       const bech32Address = 'tltc1qu78xur5xnq6fjy83amy0qcjfau8m367defyhms';
       const version1Address = tltc.canonicalAddress(bech32Address, 1);
       version1Address.should.equal(bech32Address);
       const version2Address = tltc.canonicalAddress(bech32Address, 2);
       version2Address.should.equal(bech32Address);
     });
+
+    it('non-lowercase bech32 testnet address should fail', function () {
```

Comment:
"upper case" rather than "non-lowercase"? Since we're explicitly testing upper case here, whereas non-lower could be mixed case as well (which is always invalid, maybe should be its own separate test case even).

Reply:
added more test cases to cover lowercase, uppercase and mixed case scenarios